recipes/squeegee/meta.yaml

```diff
@@ -0,0 +1,30 @@
+{% set version = "0.1.2" %}
+package:
+  name: squeegee
+  version: '{{version}}'
+source:
+  url: 'https://gitlab.com/treangenlab/squeegee/-/archive/v{{ version }}/squeegee-v{{ version }}.tar.gz'
+  sha256: 012722b7354a1b0b554ab3b9551c981b402398207b225b4dc20a9a6cb0682d4d 

```

Comment:
Try running `curl https://gitlab.com/treangenlab/squeegee/-/archive/v0.1.2/squeegee-v0.1.2.tar.gz | shasum -a 256` to get the correct hash.